src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,39 +1,80 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const filter = useAppSelector(state => state.filter);
+  const dispatch = useAppDispatch();
+
+  const handleChangeSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const status = event.target.value;
+
+    switch (status) {
+      case 'all':
+        return dispatch(filterActions.setStatus(Status.All));
+      case 'active':
+        return dispatch(filterActions.setStatus(Status.Active));
+      case 'completed':
+        return dispatch(filterActions.setStatus(Status.Completed));
+      default:
+        return 'error';
```

Comment:
what does it mean?

you can use 'all' case as default but not some random value)